src/main/java/conflux/dex/controller/ProductController.java

```diff
@@ -269,11 +271,15 @@ public Product get(@PathVariable String nameOrId) {
 	 */
 	@GetMapping
 	public ProductPagingResult list(
-			@RequestParam(required = false, defaultValue = "0") int offset, 
+			@RequestParam(required = false, defaultValue = "true") Boolean filter,
```

Comment:
Better to define the `filter` as last parameter, and filter out disabled products by default.

Reply:
ok